risc0/cargo-risczero/src/commands/new.rs

```diff
@@ -138,11 +138,10 @@ impl NewCommand {
         let guest_name = match &self.guest_name {
             Some(name) => name.clone(),
             None => {
-                eprint!(
-                    "Guest name was not supplied through the --guest-name option. Please enter\x20\
-                    package name for your template or press [enter] to use default guest package\x20\
-                    name \"method\"\n\
-                    Enter package name > "
+                eprint!("Guest name was not supplied through the --guest-name option. Please enter \
+                         package name for your template or press [enter] to use default guest package \
+                         name \"method\".\n\
```

Comment:
We just had problems with whitespace not rendering with lines ending with ` \` (see #1123), does this revision work correctly or does it display as `enterpackage` and `packagename`?

Reply:
That was the reason I changed it. Now it looks good.